### PR TITLE
0.6.4 Release notes

### PR DESCRIPTION
**Pull request by jakirkham**
Pulls together the 0.6.4 release notes from the various PRs that have been merged. Thanks everyone for your contributions. Please take a look and make sure everything looks accurate.

cc @alimanfoo @jrbourbeau @llllllllll @QuLogic

TODO:
* [ ] Unit tests and/or doctests in docstrings
* [ ] ``tox -e py37`` passes locally
* [ ] ``tox -e py27`` passes locally
* [ ] Docstrings and API docs for any new/modified user-facing classes and functions
* [ ] Changes documented in docs/release.rst
* [ ] ``tox -e docs`` passes locally
* [ ] AppVeyor and Travis CI passes
* [ ] Test coverage to 100% (Coveralls passes)

### CODE REVIEW

**Comment by jakirkham** [comment]
Just as a note, would like to address the `cpuinfo` point @QuLogic has raised before releasing. Otherwise this is basically final IMO.

File: docs/release.rst [hunk 1]
```diff
@@ -2,22 +2,61 @@ Release notes
 =============
 
 
+.. _release_0.6.4:
+
+0.6.4
+-----
+
+* Update Cython to 0.29.6.
```
Comment (jrbourbeau) [inline docs/release.rst]:
```suggestion
* Update Cython to 0.29.14.
```

File: docs/release.rst [hunk 2]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+  By :user:`John Kirkham <jakirkham>`, :issue:`168`, :issue:`177`.
```
Comment (jrbourbeau) [inline docs/release.rst]:
```suggestion
  By :user:`John Kirkham <jakirkham>`, :issue:`168`, :issue:`177`, :issue:`204`.
```

File: docs/release.rst [hunk 3]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+
+* The bundled c-blosc sources have been upgraded to version 1.17.0.
+  This fixes compilation with newer versions of gcc.
+  By :user:`Joe Jevnik <llllllllll>`, :issue:`194`
```
Comment (alimanfoo) [inline docs/release.rst]:
```suggestion
  By :user:`Joe Jevnik <llllllllll>`, :issue:`194`.
```

File: docs/release.rst [hunk 4]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+
+* Create ``.pep8speaks.yml``. By :user:`Alistair Miles <alimanfoo>`
```
Comment (alimanfoo) [inline docs/release.rst]:
```suggestion
* Create ``.pep8speaks.yml``. By :user:`Alistair Miles <alimanfoo>`.
```

File: docs/release.rst [hunk 5]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+* Simplify datetime/timedelta check.
+  By :user:`John Kirkham <jakirkham>`, :issue:`170`, :issue:`171`.
+
+* Update URL metadata for PyPI.
+  By :user:`Elliott Sales de Andrade <QuLogic>`, :issue:`178`.
+
+* Enable pytest rewriting in test helper functions.
+  By :user:`Elliott Sales de Andrade <QuLogic>`, :issue:`185`.
+
+* Rewrites the `ensure_text` implementation
```
Comment (alimanfoo) [inline docs/release.rst]:
```suggestion
* Rewrites the `ensure_text` implementation.
```

File: docs/release.rst [hunk 6]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+* Add macOS to CI.
+  By :user:`Alistair Miles <alimanfoo>`, :issue:`192`.
+
+* Fix test failures on big-endian systems.
+  By :user:`Elliott Sales de Andrade <QuLogic>`, :issue:`186`.
+
+* Use unittest.mock on Python 3.
+  By :user:`Elliott Sales de Andrade <QuLogic>`, :issue:`179`.
+
+* BLD: don't mask compile errors in setup.py.
```
Comment (alimanfoo) [inline docs/release.rst]:
```suggestion
* Don't mask compile errors in setup.py.
```

File: docs/release.rst [hunk 7]
```diff
@@ -2,22 +2,61 @@ Release notes
...
+  By :user:`Joe Jevnik <llllllllll>`, :issue:`197`
```
Comment (alimanfoo) [inline docs/release.rst]:
```suggestion
  By :user:`Joe Jevnik <llllllllll>`, :issue:`197`.
```

**Comment by jakirkham** [comment]
Thanks all! Have accepted your changes. Please let me know if there is anything else. 🙂

**Comment by jakirkham** [comment]
Thank all! 😄

**Pull request merged**